lib/web/mage/backend/suggest.js

```diff
@@ -245,6 +245,20 @@
 
                         case keyCode.ENTER:
                         case keyCode.NUMPAD_ENTER:
+                            suggestList = event.currentTarget.parentNode.getElementsByTagName('ul')[0];
```

Comment:
Is there any specific reason not to use jQuery for this purpose?

Reply:
@ishakhsuvarov you are right, I could have used jQuery, but vanillaJS does good also for things like this, and the rest of the file is retrieving elements via the same syntax, so I remained consistent.